src/shared/services/activity/activity.polling.service.ts

```diff
@@ -105,17 +119,31 @@ export class ActivityPollingService {
     })
   }
 
-  postViewActivity(caseId: string): Observable<Activity[]> {
+  public postViewActivity(caseId: string): Observable<Activity[]> {
     return this.postActivity(caseId, ActivityService.ACTIVITY_VIEW);
   }
 
-  postEditActivity(caseId: string): Observable<Activity[]> {
+  public postEditActivity(caseId: string): Observable<Activity[]> {
     return this.postActivity(caseId, ActivityService.ACTIVITY_EDIT);
   }
 
+  private init(): void {
+    this.pollConfig = {
+      interval: this.config.getActivityNexPollRequestMs(),
+      attempts: this.config.getActivityRetry(),
+      backgroundPolling: true
+    };
+    this.batchCollectionDelayMs = this.config.getActivityBatchCollectionDelayMs();
+    this.maxRequestsPerBatch = this.config.getActivityMaxRequestPerBatch();
+  }
+
+  private destroy(): void {
```

Comment:
Empty method body. Do we need the method?

Reply:
Good spot. I realised that the `stopPolling()` method did what I needed _this_ method to do... so this is redundant and should be ditched.

Reply:
Done.